app/routes/crate/owners.js

```diff
@@ -1,9 +1,13 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class OwnersRoute extends Route {
-  setupController(controller) {
-    super.setupController(...arguments);
+  @service notifications;
+  @service redirector;
```

Comment:
these service don't appear to be used. was that intentional?

Reply:
Removed.